chpl/chpl-service/src/main/resources/jobs.xml

```diff
@@ -432,7 +432,25 @@
                 </entry>
             </job-data-map>
         </job>
-
+        <!-- Chart data creator -->
```

Comment:
Also needs to be in the other jobs.xml files, otherwise it won't deploy on DEV/STG/PROD